packages/client/src/registry/searchStreams.ts

```diff
@@ -7,6 +7,13 @@ import { filter, map, unique } from '../utils/GeneratorUtils'
 import { SynchronizedGraphQLClient } from '../utils/SynchronizedGraphQLClient'
 import { Stream } from '../Stream'
 import { EthereumAddress, Logger, toEthereumAddress } from '@streamr/utils'
+import { StreamSortOptions } from '../utils/StreamSortOptions'
+import { SortDirection } from '../utils/SortDirection'
+
+export const DEFAULT_STREAM_SORT = {
```

Comment:
We could remove this and inline the value to `StreamrClient:354`. We could document there what is the default value, but maybe no need to have a separate constant which end-users could refer to?